helix-term/src/commands.rs

```diff
@@ -272,6 +272,7 @@ impl MappableCommand {
         buffer_picker, "Open buffer picker",
         jumplist_picker, "Open jumplist picker",
         symbol_picker, "Open symbol picker",
+        git_status_picker, "Open Git status picker",
```

Comment:
I think this should be called changed_file_picker and default automatically detect the VCS. Furthermore everything git specific should got into helix-vcs and be hidden behind the `DiffProvider` trait so it's easy to add support for other VCS in the future. Probably something like `changed_files(&self, rev: Option<&str>) -> Option<(Vec<PathBuf, ChsngeStatus)>`

Reply:
Ah I actually looked into the diff gutter impl and was aware of of the trait, but thought that I'm not exactly "diffing" here so decided to leave the trait along. No problem will refactor to fix this.

Reply:
Refactored everything into `helix-vcs` behind the `DiffProvider` trait. The code is much cleaner now indeed. Thanks!